security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -21,7 +21,7 @@ The following advisory is available for the {cert-manager-operator} 1.11.4:
 
 * link:https://access.redhat.com/errata/RHEA-2023:4081[RHEA-2023:4081]
 
-For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.11/[cert-manager project release notes for v1.11]. The golang version is updated to the version `1.19.10` to fix Common Vulnerabilities and Exposures (CVEs).
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.11/[cert-manager project release notes for v1.11]. The golang version is updated to the version `1.19.10` to fix Common Vulnerabilities and Exposures (CVEs). The `1.11.4` version of the {cert-manager-operator} is based on the upstream version `1.11.4`.
```

Comment:
Can you add cert-manager links? Either release tags or cert-manager.io relevant web page links would be helpful.

Reply:
Just "upstream" could be confusing for some users.